### ISSUE TITLE
chksum errors increasing on unavail device

### ISSUE THREAD

**Issue by devZer0** [openzfs/zfs#12454]
Distribution Name	| proxmox 
Distribution Version	| 6.4
Kernel Version	|  5.4.114-1-pve
Architecture	| x86_64
OpenZFS Version	|  zfs-0.8.5-pve1 , zfs-kmod-2.0.4-pve1

i have small system with 60gb ssd which got faulty. iirc, the system had that disk mirrored, but i may have removed the disk a while ago physically, but did not yet remove it from zpool configuration. 

now the remaining disk starts getting hiccup and throws errors (its simply getting old and is worn out). time for replace/reinstall. the system is unimportant, so that is not a a problem.

for my curiosity, while scrubbing the faulty ssd , i see the CKSUM counter increase for the UNAVAIL ssd 

this looks really weird

```
root@s900:~# zpool status -v 1
  pool: rpool
 state: DEGRADED
status: One or more devices could not be used because the label is missing or
	invalid.  Sufficient replicas exist for the pool to continue
	functioning in a degraded state.
action: Replace the device using 'zpool replace'.
   see: http://zfsonlinux.org/msg/ZFS-8000-4J
  scan: scrub in progress since Tue Aug  3 22:40:34 2021
	10.6G scanned at 160M/s, 7.53G issued at 113M/s, 10.6G total
	0B repaired, 70.70% done, 0 days 00:00:28 to go
config:

	NAME                                                  STATE     READ WRITE CKSUM
	rpool                                                 DEGRADED     0     0     0
	  mirror-0                                            DEGRADED     0     0     0
	    ata-KINGSTON_SV300S37A60G_50026B723107BC81-part3  DEGRADED     0     0     0  too many errors
	    8773992546633526901                               UNAVAIL      0     0 58.3K  was /dev/disk/by-id/ata-KINGSTON_SV300S37A60G_50026B723107BC67-part3

errors: No known data errors
^C
root@s900:~# zpool status -v 1
  pool: rpool
 state: DEGRADED
status: One or more devices could not be used because the label is missing or
	invalid.  Sufficient replicas exist for the pool to continue
	functioning in a degraded state.
action: Replace the device using 'zpool replace'.
   see: http://zfsonlinux.org/msg/ZFS-8000-4J
  scan: scrub repaired 0B in 0 days 00:01:35 with 0 errors on Tue Aug  3 22:42:09 2021
config:

	NAME                                                  STATE     READ WRITE CKSUM
	rpool                                                 DEGRADED     0     0     0
	  mirror-0                                            DEGRADED     0     0     0
	    ata-KINGSTON_SV300S37A60G_50026B723107BC81-part3  DEGRADED     0     0     0  too many errors
	    8773992546633526901                               UNAVAIL      0     0 70.3K  was /dev/disk/by-id/ata-KINGSTON_SV300S37A60G_50026B723107BC67-part3
```

**Comment by rincebrain** [comment]
#11629

You could cherrypick #11930 or try to get it landed in a hypothetical 2.0.6.

**Comment by behlendorf** [comment]
The fix for this is included in the 2.1.x releases.
